CRM/Core/BAO/EntityTag.php

```diff
@@ -319,7 +320,7 @@ public static function getContactTags($contactID, $count = FALSE) {
 
     if ($count) {
       $dao->fetch();
-      return $dao->cnt;
+      return (int) $dao->cnt;
```

Comment:
Because cnt is sooo much shorter than count .....

Reply:
Cuter too

Reply:
I'm fairly certain that I've seen errors in the past when trying to use `count` as a column-alias in DAO queries. Never traced the cause - just made a mental note to go along with `cnt` as the typical work-around name.

But... in the current stack, it looks like `SELECT ... AS count` works fine. So I'm +1 for the `count`...

Reply:
@totten you do know my comments about always 'reading' variables applies - there is only one way to read cnt outloud